src/migrations/1706037450493_events-delegation-types.ts

```diff
@@ -0,0 +1,28 @@
+import { ColumnDefinitions, MigrationBuilder } from "node-pg-migrate"
+import EventModel from "../back/models/Event"
+import { EventType } from "../shared/types/events"
+import { EVENT_TYPE } from "./1702322343224_create-events-table"
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+const EVENT_TYPE_OLD = 'event_type_old'
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.renameType(EVENT_TYPE, EVENT_TYPE_OLD)
+  pgm.createType(EVENT_TYPE, Object.values(EventType))
+  pgm.alterColumn(EventModel.tableName, 'event_type', {
+    type: EVENT_TYPE,
+    using: `"event_type"::text::${EVENT_TYPE}`
+  })
+  pgm.dropType(EVENT_TYPE_OLD)
+}
+
+export async function down(pgm: MigrationBuilder): Promise<void> {
+  pgm.renameType(EVENT_TYPE, EVENT_TYPE_OLD)
+  pgm.createType(EVENT_TYPE, Object.values(EventType))
+  pgm.alterColumn(EventModel.tableName, 'event_type', {
+    type: EVENT_TYPE,
+    using: `"event_type"::text::${EVENT_TYPE}`
+  })
+  pgm.dropType(EVENT_TYPE_OLD)
+}
```

Comment:
Shouldn't all this be the reverse process?

Reply:
It's both, the way up and the way down. We just need the create type command to reevaluate with the new values of the enum, and do the switcheroo

Reply:
<img width="294" alt="image" src="https://github.com/decentraland/governance/assets/2858950/bb83f4bb-bc41-4986-9747-7070fe3a8b7e">